### PR TITLE
Change current book to quick start guide

### LINKED ISSUE #261: Introduction
https://bevyengine.org/learn/book/introduction/

has

> This official book is still very incomplete. It will help you get started with the setup and learning the basics, but it does not yet cover most of Bevy's features. See the Next Steps page for links to other, more exhaustive, learning resources you can use.

"See the Next Steps" is linking to the https://bevyengine.org/learn/book/next-steps/ with

> You have reached the end of The Bevy Book!

maybe linking to https://bevyengine.org/learn/book/getting-started/ would be superior

## Context

Migrated from https://github.com/bevyengine/bevy/issues/3623 for @matkoniecz

### LINKED ISSUE #875: Change current book into quick start guide in `new-book`
Apart of #870 

As the info in the current book is still useful whilst the proper book, and quick start guide, are being worked on it should be moved to the quick start guide in the meantime. Base work for this was established in #871 as a result of resolving the merge conflicts of the current book and similarly named sections in new book.

Key things that need to be done are:
- Going thru the established quick start guide and make sure any reference to it as `The Book` or `The Bevy Book` is changed to `The Quick Start Guide` or `The Quick Start Book` (whichever you choose, please maintain consistency) if apt for the context, else some other appropriate variant like `Quick Start`, `Quick Start [Guide|Book]`, et cetera.
- In the `templates/layout/base.html` make `/learn/quick-start/` sections and pages have a header message of `Quick Start`.
- Add a link card to the learn page for the quick start guide.
- Add a link alias of `https://www.bevyengine.org/learn/book/quick-start` to the quick start guide Introduction page, even tho the quick start is currently in the learn directory. (Perhaps this is a sign that the ideal place would be to put quick start is in the book, but that significantly complicates hiding the book while it is worked on, so probably not).

### PR DESCRIPTION
**Pull request by TrialDragon**
Changes references inside the quick start guide from book to quick start guide, fixes links inside of the quick start guide that used to go to the book. It also clarifies and fixes the last section in next steps.

Closes #875 
Fixes #261

### CODE REVIEW

**Review by alice-i-cecile — APPROVED** [review]

File: content/learn/links.toml
Comment (hymm) [inline content/learn/links.toml:7]: should the book link above this line be hidden? it's 404ing

File: content/learn/quick-start/introduction/_index.md
Comment (hymm) [inline content/learn/quick-start/introduction/_index.md:39]: there's a `learn/book/contributing` link on line 29 that needs to be updated in this file

**Review by hymm — COMMENTED** [review]
commented about a couple links that 404'ed. Should be ready to go after those are fixed.

**Comment by hymm** [comment]
found one more 404 in `getting-started\setup\_index.md` line 205 "troubleshooting section"

File: content/learn/links.toml
Comment (TrialDragon) [reply content/learn/links.toml:7]: That is removed in #879 which has been merged already.

**Comment by hymm** [comment]
some of the blog posts link to `/learn/book/introduction` should there be an alias for that to `/learn/getting-started/introduction`?

**Comment by TrialDragon** [comment]
> some of the blog posts link to `/learn/book/introduction` should there be an alias for that to `/learn/getting-started/introduction`?

The plan is to change all of those in #878 I believe. Probably switch them to the `/learn/book/quick-start` link.

**Comment by hymm** [comment]
> The plan is to change all of those in https://github.com/bevyengine/bevy-website/issues/878 I believe. Probably switch them to the /learn/book/quick-start link. But if it is desired I can push it up to this PR.

I don't have the data to say whether we should clean those up or not. Depends on how much traffic those pages are getting. If I had to guess it'd probably be ok to let the be broken for a day or two, but hard to say for sure.

**Comment by TrialDragon** [comment]
> I don't have the data to say whether we should clean those up or not. Depends on how much traffic those pages are getting. If I had to guess it'd probably be ok to let the be broken for a day or two, but hard to say for sure.

I mean, it's not touching the deployed site afaik. This PR is only targeting `new-branch`, not `main` This _will_ be fixed before we merge into main.

**Review by hymm — APPROVED** [review]
looks good other than the broken links to `/learn/book/introduction` in the news posts. @alice-i-cecile your call whether we should fix it in this pr or a follow up.

**Comment by alice-i-cecile** [comment]
Let's fix those broken links here.

**Pull request merged**
